tokio/src/runtime/context.rs

```diff
@@ -84,17 +85,22 @@ cfg_rt! {
     }
 
     #[derive(Debug)]
+    #[must_use]
     pub(crate) struct SetCurrentGuard {
         old_handle: Option<scheduler::Handle>,
         old_seed: RngSeed,
     }
 
     /// Guard tracking that a caller has entered a runtime context.
+    #[must_use]
     pub(crate) struct EnterRuntimeGuard {
         pub(crate) blocking: BlockingRegionGuard,
+        #[allow(dead_code)] // Only tracking the guard.
+        pub(crate) handle: SetCurrentGuard,
```

Comment:
```suggestion
        pub(crate) _handle: SetCurrentGuard,
```

Reply:
I don't like `_` personally. I prefer the annotation

Reply:
Well, "I don't like it" because, after talking with quite a few new rust users, it confuses them without any real value. The annotation makes it clear when reading, that the value is intended to be unused. Also, the pattern is rare enough that we don't need an obscure short hand.